ogr/services/github.py

```diff
@@ -43,8 +43,8 @@ def create_from_remote_url(cls, remote_url, **kwargs):
     @staticmethod
     def is_fork_of(user_repo, target_repo):
         """ is provided repo fork of gh.com/{parent_repo}/? """
-        return user_repo.fork_create and user_repo.parent and \
-               user_repo.parent.full_name == target_repo
+        return user_repo.fork_create and user_repo.get_parent and \
```

Comment:
isn't get_parent meant to be a method?

Reply:
Thanks, good catch. Boolean methods did not rise error.